test/cri-containerd/policy_test.go

```diff
@@ -84,22 +83,32 @@ func alpineSecurityPolicy(t *testing.T, policyType string, allowEnvironmentVaria
 		}
 	}
 
-	containers := append(defaultContainers, alpineContainer)
-	policyString, err := securityPolicyFromContainers(policyType, true, containers, allowEnvironmentVariableDropping)
-	if err != nil {
-		t.Fatalf("failed to create security policy string: %s", err)
-	}
-	return policyString
+	containers := append(helpers.DefaultContainerConfigs(), alpineContainer)
+	return policyFromOpts(
+		t,
+		policyType,
+		securitypolicy.WithContainers(containers),
+		securitypolicy.WithExternalProcesses(defaultExternalProcesses),
+		securitypolicy.WithAllowUnencryptedScratch(true),
+		securitypolicy.WithAllowEnvVarDropping(allowEnvironmentVariableDropping),
+		securitypolicy.WithAllowRuntimeLogging(true),
+		securitypolicy.WithAllowPropertiesAccess(true),
+		securitypolicy.WithAllowDumpStacks(true),
+	)
 }
 
 func sandboxRequestWithPolicy(t *testing.T, policy string) *runtime.RunPodSandboxRequest {
 	t.Helper()
+	securityHardware := false
+	if *flagSevSnp {
+		securityHardware = true
+	}
```

Comment:
cant this be `securityHardware := *flagSevSnp`? or just use `*flagSevSnp` directly?

Reply:
lol yeah. definitely